.gitignore

```diff
@@ -11,14 +11,15 @@
 
 # Generated backend configuration
 backend-config.hcl
+*backend.tf
 
 # Key files
 *.pem
 
 .DS_Store
 
 # Environment files
-*.env
+*.env*
```

Comment:
Note that this will also exclude files like `.env.example` if we decide to have an example env file. Is this the desired behaviour?

Reply:
updated